core/src/main/java/org/apache/calcite/sql2rel/StandardConvertletTable.java

```diff
@@ -958,7 +960,17 @@ private static List<Integer> elseArgs(int count) {
 
   private static List<RexNode> convertOperands(SqlRexContext cx,
       SqlCall call, SqlOperandTypeChecker.Consistency consistency) {
-    return convertOperands(cx, call, call.getOperandList(), consistency);
+    List<SqlNode> operandList;
+    if (call.getOperator() instanceof SqlTableFunction) {
+      // skip set semantic table node of table function
```

Comment:
I see too many instance of for `SqlTableFunction`, can it has its own SqlKind ? Or can we use the COLLECTION_TABLE as its sql kind ?